packages/adapter-fauna/src/index.ts

```diff
@@ -0,0 +1,249 @@
+import { LuciaError, generateRandomString } from "lucia-auth";
+import type { Adapter } from "lucia-auth";
+import faunadb from "faunadb";
+import {convertUserResponse} from "./utils.js";
```

Comment:
Did you format the code? Shouldn't it be `{ convertUserResponse }`? Also, some lines are missing `;`s.

Reply:
I ran the prettier config now